awx/sso/pipeline.py

```diff
@@ -255,17 +255,18 @@ def _check_flag(user, flag, attributes, user_flags_settings):
             # If so, check and see if the value of the attr matches the required value
             attribute_value = attributes.get(attr_setting, None)
             if isinstance(attribute_value, (list, tuple)):
```

Comment:
Do we have documentation that attributes will always be a list? If so we don't need this if condition. If not, we should also account for strings here.